src/main/java/com/maxmind/geoip2/record/Traits.java

```diff
@@ -45,29 +64,19 @@
      * This attribute is only available from the City and Insights web
      * service end points.
      */
+    @JsonProperty("autonomous_system_number")
     public Integer getAutonomousSystemNumber() {
         return this.autonomousSystemNumber;
     }
 
-    /**
-     * @return The organization associated with the registered <a
-     * href="http://en.wikipedia.org/wiki/Autonomous_system_(Internet)"
-     * >autonomous system number</a> for the IP address. This attribute
-     * is only available from the City and Insights web service end
-     * points.
-     */
-    public String getAutonomousSystemOrganization() {
-        return this.autonomousSystemOrganization;
-    }
 
```

Comment:
The order of the getters control the how the order of the keys in the actual json being generated. Without re-ordering the `JsonTest` test failed. As I didn't want to change any existing test, I changed the order of the getters instead of the test.